### ISSUE TITLE
Manifest Vendor / Progression data contain invalid properties

### ISSUE THREAD

**Issue by sgtfrankieboy** [Bungie-net/api#64]
Each entry in the `DestinyVendorDefinition` and `DestinyProgressionDefinition` table have next to the `displayProperties` a property named `BungieNet.Engine.Contract.Destiny.World.Definitions.IDestinyDisplayDefinition.displayProperties`.

The duplicate entry contains the exact same information as the regular one. I think this is happening because they both have a DisplayProperty that is overloading the standard one.

**Comment by vthornheart-bng** [comment]
Indeed, it's true - good eye!  It's a bit of trickiness I'm doing regarding subclasses of DestinyDisplayPropertiesDefinition and maintaining compatibility with an interface that we use internally... I didn't notice they were getting output!  I'll have to look into that and see if I can suppress them.

**Comment by vthornheart-bng** [comment]
(phew, we've got so many bugs to resolve once I catch up on launch-related issues)
